### PR TITLE
add .yeezy

### PR DESCRIPTION
**Pull request by lalanza808**

### CODE REVIEW

File: trifling.py [hunk 1]
```diff
@@ -889,3 +889,11 @@ def weather(bot, trigger):
 
     except:
         bot.say("The earth is on fire 🌎🔥")
+
+@sopel.module.commands('yeezy')
+def yeezy(bot, trigger):
+    try:
+        res = requests.get('https://api.kanye.rest')
```
Comment (sanderfoobar) [inline trifling.py]: - add a timeout
- add an user agent
- check for 2XX status code
- check response type
- validate json schema

```python3
headers = {"User-Agent": "Wow!"}
resp = requests.get('https://api.kanye.rest', headers=headers, timeout=3)
resp.raise_for_status()
blob = resp.json()
if "quote" not in blob or not isinstance(blob["quote"], str):
    raise Exception("malformed response")
bot.say(blob["quote"])
```

Comment (lalanza808) [reply trifling.py]: sacre bleu! am I in a code review right now?!

Comment (sanderfoobar) [reply trifling.py]: I'm afraid so!!

Comment (lalanza808) [reply trifling.py]: done, sir.

**Comment by lalanza808** [comment]
sorry, my editor thought the whitespaces here were wack

**Comment by sanderfoobar** [comment]
w0w such enterprise :-)

**Review by sanderfoobar — APPROVED** [review]

**Pull request merged**